api/bases/test.openstack.org_ansibletests.yaml

```diff
@@ -115,6 +115,13 @@ spec:
                 description: OpenStackConfigSecret is the name of the Secret containing
                   the secure.yaml
                 type: string
+              parallel:
+                default: false
+                description: By default test-operator executes the test-pods sequentially
+                  if multiple instances of test-operator related CRs exist. If you
+                  want to turn off this behaviour then set this option to true. TODO(kstrenko)
+                  This is a placeholder, logic is not implemented
```

Comment:
I'm just checking. Is this still true?

Reply:
This is added for efficiency -> as far as I know AnsibleTest CR does not have parallel option yet, but will have in future. Thanks to adding this placeholder we can use the same logic (for creating multiple PVCs) as in Tempest and Tobiko CR and not open a ticket on adding this later.

If we do not want to add placeholder I can edit the change as it is in Horizon CR, but when the parallel option will be added later, we need to change the logic for creating multiple PVCs separately.

Maybe I overexplained it, but let me know what you think:)

Reply:
> If we do not want to add placeholder I can edit the change as it is in Horizon CR, but when the parallel option will be added later, we need to change the logic for creating multiple PVCs separately.

As discussed, let's use the same approach as with Horizon CR. The danger of this current approach is that we are exposing a new parameter to the user that in the end does nothing. Thank you for the explanation @kstrenkova!